### PR TITLE
refactor(core): PHP 8.0 support started

### PR DESCRIPTION
**Pull request by Guikingone**

### CODE REVIEW

**Comment by jmsche** [comment]
Congrats! :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ CHANGELOG
 
  * Task notifications added (see https://github.com/Guikingone/SchedulerBundle/pull/1)
  * Task & Worker lifecycle logs added (see https://github.com/Guikingone/SchedulerBundle/pull/5)
+ * PHP 7.2 & 7.3 support dropped ()
+ * PHP 8.0 support added ()
```
Comment (jmsche) [inline CHANGELOG.md:15]:
```suggestion
 * PHP 7.2 & 7.3 support dropped (see https://github.com/Guikingone/SchedulerBundle/pull/13)
 * PHP 8.0 support added (see https://github.com/Guikingone/SchedulerBundle/pull/13)
```

File: src/Runner/CallbackTaskRunner.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ public function run(TaskInterface $task): Output
 
             $task->setExecutionState(TaskInterface::SUCCEED);
 
-            return new Output($task, trim(!is_string($output) ? (string) $output : $output));
+            return new Output($task, trim(is_string($output) ? $output : (string) $output));
```
Comment (jmsche) [inline src/Runner/CallbackTaskRunner.php:42]:
```suggestion
            return new Output($task, trim((string) $output));
```
Not sure about this one

**Review by jmsche — CHANGES_REQUESTED** [review]

**Review by jmsche — APPROVED** [review]

**Comment by Guikingone** [comment]
Thanks @jmsche for the support and the time 🙂  👍🏻

**Pull request merged**